phoenicis-dist/src/scripts/phoenicis-create-package.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+## Dependencies
+# Linux:
+# - fakeroot
+#
+
+VERSION="$1"
```

Comment:
Can't we do this in maven?

Reply:
Calling the script from maven, maybe. Doing what it does, not easily except if we create a plugin (which would be a cool open source contribution though)

Reply:
Is it because we basically have 2 packaging stages (jpackager and then deb/dmg etc)?

Reply:
jpackager does pretty much everything for Linux including generating a .deb archive, except that it does not install scripts in /usr/bin. So far, everything is installed in /opt. 

This is not ideal but good enough for testing the concept. If we find out that the approach are good for you, I'll improve the .deb structure so that it it matches POLv4 one

Reply:
Concerning .osx, I did not find how to put the .app in a dmg but this can also be easily scripted

Reply:
Finally, there are no maven plugin for jpackager afaik

Reply:
I'm fine with separating the build and the packaging but there should be a general concept. For example, we now have information about the .deb in the control file and also in the script. This doesn't make a lot of sense to me.